src/hyp3lib/asf_geometry.py

```diff
@@ -820,7 +820,7 @@ def list2shape(csvFile, shapeFile):
     values = []
     field['name'] = 'granule'
     field['type'] = ogr.OFTString
-    field['width'] = 254
+    field['width'] = 254  # type: ignore [assignment]
```

Comment:
What was the error here? If `mypy` is confused about the key and value types for the `field` dict, it might be solved by changing the `field` definition to `field: dict = {}` so that it gets typed as `dict[Any, Any]`.

Reply:
I'll open a PR for this.